vtex-apps/return-app
# Fix: Merge fixes for rootPath and double returns

#### What this PR changes
On this PR we integrated the latest changes from the [fix/binding-rootPath](https://github.com/filafb/return-app/tree/fix/binding-rootpath) and the [latest fixes](https://github.com/vtex-apps/return-app/pull/40) for the app.

#### How to test it:
The app is linked on two workspaces:
1. To test the `getProfile`  on a store with multiple bindings, you can inspect the requests (network tab) on [this workspace](https://www.yamamay.com/it_it/account?workspace=returnapptest#/my-returns).
The requests should look like this: `https://www.yamamay.com/it_it/no-cache/profileSystem/getProfile`

2. To test the fix for double returns, you can create an order in [this workspace](https://returnapptest--ametllerorigenqa.myvtex.com). The orders should be invoiced.
After the order has been invoiced, you can go to [`My account/My returns`](https://returnapptest--ametllerorigenqa.myvtex.com/account?__bindingAddress=www.ametllerorigenqa.com/#/my-returns) page and make a return request. You should not be able to request double returns for a given product's item.
